app/lib/actions/candlepin/reindex_pool_subscription_handler.rb

```diff
@@ -45,23 +45,25 @@ def handle(message)
       private
 
       def reindex_or_unindex_pool(pool_id)
-        ::Katello::Pool.find_pool(pool_id)
-        reindex_pool(pool_id)
+        pool = ::Katello::Pool.candlepin_data(pool_id)
+        ::Katello::Pool.find_by_cp_id(pool_id).import_data
```

Comment:
In other conversions, similar to this function, we have changed our word to 'import' over 'index'. What do you think about updating the logic in here to reflect that? Given that functions like `reindex_or_unindex_pool` and `reindex_pool` exist?